internal/live/live.go

```
@@ -0,0 +1,515 @@
+
+// maybeGunzip decompresses frames whose first two bytes are the gzip magic
+// (the channel negotiates compress=gzip); uncompressed frames pass through.
+func maybeGunzip(b []byte) ([]byte, error) {
+	if len(b) < 2 || b[0] != 0x1f || b[1] != 0x8b {
+		return b, nil
+	}
+	zr, err := gzip.NewReader(bytes.NewReader(b))
+	if err != nil {
+		return nil, err
+	}
+	defer zr.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `zr.Close` is not checked

(errcheck)